File: WPF客户端/AnnHome/AnnHome/Entity/Categories.cs
using System;
using System.Collections.Generic;

namespace AnnHome.Entity
{
    [Serializable]
    internal class Categories:List<Object>
    {
        private int id;
        private string slug;
        private string title;
        private string description;
        private int parent;
        private int post_count;

        public int Id
        {
            get
            {
                return id;
            }

            set
            {
                id = value;
            }
        }

        public string Slug
        {
            get
            {
                return slug;
            }

            set
            {
                slug = value;
            }
        }

        public string Title
        {
            get
            {
                return title;
            }

            set
            {
                title = value;
            }
        }

        public string Description
        {
            get
            {
                return description;
            }

            set
            {
                description = value;
            }
        }

        public int Parent
        {
            get
            {
                return parent;
            }

            set
            {
                parent = value;
            }
        }

        public int Post_count
        {
            get
            {
                return post_count;
            }

            set
            {
                post_count = value;
            }
        }
    }
}

File: Android客户端/AnnHome/app/src/main/java/entity/Categories.java
package entity;

/**
 * Created by mrpan on 15/11/17.
 */
public class Categories {
    private int id;
    private String slug;
    private String title;
    private String description;
    private int parent;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getPost_count() {
        return post_count;
    }

    public void setPost_count(int post_count) {
        this.post_count = post_count;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getParent() {
        return parent;
    }

    public void setParent(int parent) {
        this.parent = parent;
    }

    private int post_count;
}
